packages/apps/paghiper/package.json

```
@@ -0,0 +1,34 @@
+{
+  "name": "@cloudcommerce/app-paghiper",
+  "type": "module",
+  "version": "0.0.0",
+  "description": "E-Com Plus Cloud Commerce app to integrate PagHiper",
+  "main": "lib/paghiper.js",
+  "exports": {
+    ".": "./lib/paghiper.js",
+    "./events": "./lib/paghiper-webhook.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ecomplus/cloud-commerce.git",
+    "directory": "packages/apps/paghiper"
+  },
+  "author": "E-Com Club Softwares para E-commerce <ti@e-com.club>",
+  "license": "Apache 2.0 with Commons Clause",
+  "bugs": {
+    "url": "https://github.com/ecomplus/cloud-commerce/issues"
+  },
+  "homepage": "https://github.com/ecomplus/cloud-commerce/tree/main/packages/apps/paghiper#readme",
+  "scripts": {
+    "build": "sh ../../../scripts/build-lib.sh"
+  },
+  "dependencies": {
+    "@cloudcommerce/api": "workspace:*",
+    "@cloudcommerce/firebase": "workspace:*",
+    "axios": "^1.2.2",
+    "firebase-admin": "^11.4.1"
+  },
+  "devDependencies": {
+    "@cloudcommerce/types": "workspace:*"
+  }
```

```suggestion
  "devDependencies": {
    "@cloudcommerce/types": "workspace:*",
    "@firebase/app-types": "^0.9.0"
  }
```

Toda vez tá faltando isso aí e dá erro de peer deps no `pnpm i` 🙃

---

Aí eu adiciono depois tipo https://github.com/ecomplus/cloud-commerce/commit/c0109f0837f00105189194bc6a100fc9ee318921 😄